src/compiler/diagnosticMessages.json

```diff
@@ -1396,6 +1396,10 @@
         "category": "Error",
         "code": 2415
     },
+    "Class member of type '{0}' is not assignable to base class member of type '{1}'.": {

```

Comment:
i would add the member name as well..

Reply:
The error message before was helpful too. i wounder if we should make this an elaboration, e.g.:
```shell
Class '{0}' incorrectly extends base class '{1}'.
   member '{2}' of type '{3}' is not assignable to base class member of type '{3}'.
```

Reply:
also pinging @DanielRosenwasser

Reply:
With a little more in-person deliberation, I've changed it to the following:
```
Property 'bar' in type 'B' is not assignable to the same property in base type 'Base'.
   Type 'string' is not assignable to type 'Base'.
```